### ISSUE TITLE
Make add-on periods expire once the main product expires

### ISSUE THREAD

**Issue by tobias992** [boxbilling/boxbilling#809]
- Set the addon period to the same time as the product (others make no sense)

**Comment by John-S4** [comment]
I'm not sure I understand, why do others make no sense?

**Comment by tobias992** [comment]
Because the Adons are related to a Product. Example:

Product with a 3 Month term. In this case it doesnt make sense to sell a Addon for this Product with a term of 6 or 12 Month. It must have the same term and also the same renew date.

**Comment by John-S4** [comment]
Agreed it makes no sense for it to have a longer term, but there are possible reasons for it to be able to have a shorter one.
